tangelo/linq/translator/translate_braket.py

```diff
@@ -99,3 +116,40 @@ def translate_braket(source_circuit):
         else:
             raise ValueError(f"Gate '{gate.name}' not supported on backend braket")
     return target_circuit
+
+
+def translate_c_from_braket(source_circuit) -> Circuit:
```

Comment:
I'd rather you do not document type that half-ass it. You haven't done it for any other function, and you're not doing it for the argument on the same line.

I think however we should have a conversation about types in Python as a group

Reply:
This was written from the already reviewed function in `translate_qiskit`. Just to be consistent, I removed that ("-> Circuit") everywhere in the translator module.